src/components/TaskRunsCard/index.jsx

```diff
@@ -316,7 +325,10 @@ export default class TaskRunsCard extends Component {
               </ListItem>
               <Collapse in={showArtifacts} timeout="auto">
                 <List component="div" disablePadding>
-                  <ListItem component="div" disableGutters>
+                  <ListItem
+                    className={classes.artifactsListItemContainer}
+                    component="div"
```

Comment:
This `div` should already be `display: block`. Can you confirm if a `div` is rendered here?

Reply:
Yes, it is rendered as a div but the element has styles corresponding to a list item. The `display` property is not set.